README.md

```diff
@@ -80,6 +80,13 @@ To run tests
  $ npm test -- -u --coverage
 ```
 
+### Creating a [Sass](https://sass-lang.com/) build
+```
+- To compile Sass to CSS, run the command below.
+
+$ npm run build:css && eslint ./src
+```
```

Comment:
You can also add the command `npm run watch:css` to have automated sass compilation when one is working on a task.

Reply:
Thanks, @Duncanian for the tip. I will surely add that.